cloud-sql/mysql/client-side-encryption/noxfile.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
```

Comment:
Please remove this noxfile - it will be auto generated from the `noxfile-template.py` file found in the root directory. If you need to make changes to any of the default values (which, at a quick glance, I think you are using type hints so you need to make sure [enforce type hinting is set to true](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/8b72c12d77fe68592552a36f673cb0d9d9806874/dataflow/gpu-workers/noxfile_config.py#L30)) , you can add a `noxfile_config.py`. Please ping me if you need any help with that.

Reply:
Question- do we require type hints for tests as well as samples? I ended up getting a lot of type hint errors for the tests when I turned the flag for type hints on.